scripts/analysis/main.py

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jul 19 13:57:23 2023
+
+The main script for running the GARDN-M model.
+
+@author: oaknelson
+"""
+
+import git
+import pandas as pd
+import os
+import json
+import numpy as np
+
+# establish relative directories
+repo = git.Repo('.', search_parent_directories=True)
+os.chdir(repo.working_tree_dir)
```

Comment:
Is there a smarter way to handle directories?

Reply:
Perhaps use [`pathlib`](https://docs.python.org/3/library/pathlib.html), and in particular `pathlib.Path`?